src/app/modules/angular-slickgrid/models/gridOption.interface.ts

```diff
@@ -49,6 +49,9 @@ export interface GridOption {
   /** Defaults to false, when enabled will give the possibility to edit cell values with inline editors. */
   editable?: boolean;
 
+  /** option to intercept edit commands and implement undo support. Call command.execute() to finish edit operation*/
+  editCommandHandler?: (row: any, column: any, command: any) => void;
```

Comment:
I think there's a mistake on the first argument, from the 6pac [undo demo - queueAndExecuteCommand](https://github.com/6pac/SlickGrid/blob/master/examples/example3b-editing-with-undo.html#L84) the function signature is the following
```javascript
function queueAndExecuteCommand(item, column, editCommand) { }
```
I tried it locally and it does return the item data. 

Also please use the types (TypeScript) when they are known. 
So at the end I think it should be
```javascript
editCommandHandler?: (item: any, column: Column, command: any) => void;
```
We could also create a `EditCommand` type in the future, since it's probably also a know list of arguments.

Reply:
I actually found the official [wiki - editCommandHandler](https://github.com/6pac/SlickGrid/wiki/Writing-custom-cell-editors#intercepting-cell-edits) which does say the 1st argument is `item`. From that doc, we can also see that the `editCommand` (last argument) is also a known type, so we should create a new type for that, to create a new type, simply create a new interface and put it in the [angular-slickgrid/models/](https://github.com/ghiscoding/Angular-Slickgrid/tree/master/src/app/modules/angular-slickgrid/models) folder. I can do that part if you are confused about it.

Reply:
You're right 'row' is confusing name (initially I meant rowData but named it as 'row').
Good idea! I'll create EditCommand type and will rename 'row' to 'item'. Thanks for the clarification.